packages/@okta/vuepress-site/guides/sign-into-spa/section09/index.md

```diff
@@ -1,25 +1,21 @@
 ---
 title: Require Authentication
 ---
-## Require Authentication
 
-An important decision regarding your authentication requirements is which parts of your application require the user to be authenticated. You can require authentication across the entire app or just require it for particular routes or controllers. There is also the issue of what sort of app behavior you want for unauthenticated users (users who aren't signed in).
+In most applications, you'll want to prevent the user from accessing certain routes or sections unless they are signed in. You can require authentication across the entire app, or just require it for particular routes or controllers. Any route that doesn't specifically require authentication will be accessible without signing in (also called anonymous access).
 
-### Require Authentication for Everything
-
-For some applications, you want to require the user to be authenticated (signed in) for all routes.
-
-<StackSelector snippet="reqautheverything"/>
+It's important to note that protecting routes in your single-page app does **not** prevent the user from accessing those parts of your application. After all, it's JavaScript running in the browser, and anyone could open the browser's developer tools and change things! Protecting routes provides a consistent and good experience for your users. The real security enforcement must be done in the API your single-page app calls (see [Use the Access Token](todo)). <!-- link to Protect your API endpoints guide -->
```

Comment:
It's important to note that protecting routes in your single-page app does **not** prevent the user from accessing those parts of your application. After all, it's JavaScript running in the browser and anyone could open the browser's developer tools and change things! Protecting routes provides a consistent and good experience for your users. The real security enforcement must be done in the API that your single-page app calls (see [Use the Access Token](todo)). <!-- link to Protect your API endpoints guide -->